packages/client-react-streaming/package-shape.json

```diff
@@ -26,12 +28,37 @@
       "RemoveMultipartDirectivesLink",
       "SSRMultipartLink",
       "WrapApolloProvider",
-      "resetApolloSingletons"
+      "resetApolloSingletons",
+      "built_for_ssr"
+    ],
+    "edge-light,worker,browser": [
```

Comment:
This is the order of conditions that Next.js is using, so we check for that explicitly.

See 

https://github.com/vercel/next.js/blob/bbd79ac9977e03010e09ff90d933f2ec614b031f/packages/next/src/build/webpack-config.ts#L96-L103